README.md

```diff
@@ -51,13 +51,13 @@ Substrate Documentation:
 Start chain and connect to alice bootnode:
 
 ```
-docker run -p 9944:9944 kiltprotocol/mashnet-node ./start-node.sh --connect-to alice
+docker run -p 9944:9944 kiltprotocol/mashnet-node --chain kilt-testnet --bootnodes /dns4/bootnode-alice.kilt-prototype.tk/tcp/30333/p2p/12D3KooWPuXafPUY8E8zo7m4GuWkgj9ByfsanrNUznZShBgJrW4A
```

Comment:
Can we add the bootnodes into the chainspec, so that the user does not have to specify them himself?

Reply:
fixed in 76700c1